web/src/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.jpg';
```

Comment:
What does this do? is it supposed to by a .ts file extension?

Reply:
this is how pictures are able to be included in the webpack bundle. I got this from https://stackoverflow.com/questions/52759220/importing-images-in-typescript-react-cannot-find-module because other methods weren't working

Reply:
guess I didn't need this after all. Removed